### DISCUSSION TITLE
[client-preset] How to reduce bundle size with Rollup?

### DISCUSSION THREAD

**Discussion by abdonrd** [dotansimha/graphql-code-generator#8877] [Q&A]
Hello!

In the preset-client docs there is a Babel plugin to reduce the bundle size:
https://the-guild.dev/graphql/codegen/plugins/presets/preset-client#reducing-bundle-size-babel-plugin
https://github.com/dotansimha/graphql-code-generator/blob/master/packages/presets/client/src/babel.ts

We are using Vite/Rollup.

How we can use it?

Thanks!

**Comment by abdonrd** [comment]
Any help here? Thanks!

**Comment by n1ru4l** [comment]
You can use the swc plugin https://the-guild.dev/graphql/codegen/plugins/presets/preset-client#swc-plugin

**Reply by abdonrd** [reply]
@n1ru4l How do we use it? Because we don't use SWC or React. We use Vite and Vue.

**Reply by n1ru4l** [reply]
I don't use Vue. There is a SWC plugin for vite as far as I know 😅

**Reply by abdonrd** [reply]
The Vite SWC plugin looks like is just for React.

**Reply by n1ru4l** [reply]
Then it seems like you might be stuck with the babel plugin, or maybe you can ask vite maintainers how to use swc plugins with vue.js?
